README.md

```diff
@@ -102,6 +102,11 @@ and run:
 npm install
 ```
 
+* Note: package-lock.json file was generated by Node version 8 and npm version 6.4.1. Although the app supports Node versions 6 and 8 as well as multiple versions of npm, we should always use those versions when installing the package-lock.json, unless decided to upgrade those, and in this case the readme should be upadted as well. If using a different version for node OR npm, please delete the package-lock.json file prior installing the node packages and also prior pushing to github.
```

Comment:
* "updating package-lock.json" instead of "installing the package-lock.json"
* "unless it is decided"
* typo: "upadted"
* "prior to installing node packages" instead of "prior installing the node packages"
* "prior to pushing to github"